.gitignore

```diff
@@ -2,12 +2,15 @@
 __pycache__/
 *.py[cod]
 
+*.DS_Store*
```

Comment:
Could you move this to a separated commit, please?

Reply:
Done !